components/ProgressView/src/private/MDCProgressView+MotionSpec.h

```diff
@@ -0,0 +1,27 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import "MotionInterchange.h"
+
+struct MDCProgressViewMotionSpec {
+  MDMMotionTiming changeProgress;
+  MDMMotionTiming visibility;
```

Comment:
I know these are synonyms with hidden but perhaps it is better to match exactly. I would change this to "hidden." 
Otherwise we should probably comment:
"controls the pace of animation when hiding/showing the component"